src/test/java/Homework18.java

```diff
@@ -0,0 +1,28 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest{
+
+    @Test
+    public void playASongTest() {
+        login("burtoiuionut@ymail.com", "te$t$tudent");
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/216771291-aa2ebacd-0c46-4135-84db-258f61b07d13.png)

Reply:
on my laptop, I don't have any errors 

<img width="883" alt="Screenshot 2023-02-05 at 11 49 23 PM" src="https://user-images.githubusercontent.com/121268896/216886202-757c85ed-f698-4860-95c2-7eedb16f9000.png">

Reply:
<img width="988" alt="Screenshot 2023-02-05 at 11 52 14 PM" src="https://user-images.githubusercontent.com/121268896/216886488-0fb3d386-e4f7-4d0a-966d-79f6a94b481d.png">